src/pages/[platform]/build-a-backend/auth/multi-step-sign-in/index.mdx

```diff
@@ -4,7 +4,7 @@ export const meta = {
   title: 'Multi-step sign-in',
   description:
     'Use Amazon Cognito Auth plugin to complete a multi step authentication flow',
-  platforms: ['android', 'swift', 'flutter']
+  platforms: ['android', 'swift', 'flutter', 'react', 'nextjs', 'javascript', 'react-native', 'vue', 'angular']
```

Comment:
I'm hesitant to open this page up since it breaks the sidebar structure https://github.com/aws-amplify/docs/pull/7456/files#diff-ad6ae68fec71491f39c0ea656db38a9c31fc72ce1117c3e894077a879b2cddfe

Reply:
the content is fine, but can we `git mv` this into the "connect your frontend" section?

Reply:
thanks, it's been moved 👍